WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2909,12 +2909,16 @@ static GSourceFuncs _handlerIntervention =
                     webkit_memory_pressure_settings_free(memoryPressureSettings);
                 }
 #endif
+                string persistentPath = _service->PersistentPath();
+                string volatilePath = _service->VolatilePath();
```

Comment:
Do we need to assign string variables or just use _service->PersistentPath() and volatilePath as parameters? @MFransen69 what do you think?

Reply:
At first, I wanted to use `_service->PersistentPath()` and `_service->VolatilePath()` as parameters, but then I thought maybe it's better to keep it consistent with other parameters, so I assigned them to these string variables. Would it perhaps be optimized away by the compiler either way?

Reply:
I do understand the consistency concern (and I'm really fond of consistency :) ) but for the other variables it seems to make sense as the variable is actually changed from the config but here that is not the case and it is now a needless copy although the optimizer might even fix this for you :) (and std::string it is not refcounted as some sometimes think). So I would just use them directly.  (my two cents)